packages/loader/container-loader/src/container.ts

```diff
@@ -874,7 +874,9 @@ export class Container
 							: this.deltaManager?.lastMessage?.clientId,
 					dmLastMsgClientSeq: () => this.deltaManager?.lastMessage?.clientSequenceNumber,
 					connectionStateDuration: () =>
-						performance.now() - this.connectionTransitionTimes[this.connectionState],
+						// TODO why are we non null asserting here?
+						// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
I am not in favor of putting these non-null assertions disables everywhere in the code. We should find another way to not do this.

Reply:
For the initial checkin we're preferring to avoid code changes in most cases, to reduce risk of regression (and thereby risk of needing to revert large changes).  That generally means non-null assertions, which then need lint disables.

Agree that many of these can be better solved by actual code changes, but as a staging mechanism it's better to do these disables for now and followup with the cleanup.

Reply:
ok.